CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## 0.26.0 - 2025-04-02
+This release enables the creation of web apps with a unique default hostname to avoid a high-severity threat of subdomain takeover.
+The web app basic create will now default to a `tenant` scoped hostname uniqueness, while the web app advanced create will allow
+the choice of either `tenant` (tenant reuse) or `global` (no reuse) level uniqueness scope.  For full context on this topic,
+please refer to this [post](https://techcommunity.microsoft.com/blog/appsonazureblog/public-preview-creating-web-app-with-a-unique-default-hostname/4156353).
```

Comment:
You should make this an aka.ms link. Never know if it'll go down or whatever.

Reply:
Oh right, good call